core/lib/libtomcrypt/acipher_helpers.h

```diff
@@ -28,3 +33,14 @@ static inline TEE_Result convert_ltc_verify_status(int ltc_res, int ltc_stat)
 		return TEE_ERROR_GENERIC;
 	}
 }
+
+#ifdef CFG_CRYPTOLIB_NAME_tomcrypt
```

Comment:
Is this conditional needed?

Reply:
Yes, because when it is not defined (i.e., MBed TLS is used instead of LTC), `libtomcrypt/` is still used partially (for some algorithms), and this file gets included, but not all LTC typedefs get defined. In this case, `ecc_point` for instance is unknown.

Reply:
OK, thanks.